rotkehlchen/chain/evm/decoding/weth/constants.py

```diff
@@ -0,0 +1,22 @@
+import typing
+from typing import Final
+
+from rotkehlchen.constants.assets import A_WETH, A_WETH_ARB, A_WETH_BASE, A_WETH_OPT, A_WETH_SCROLL
+from rotkehlchen.types import SUPPORTED_CHAIN_IDS, ChainID
+
+CPT_WETH: Final = 'weth'
+CHAINS_WITHOUT_NATIVE_ETH: Final = {ChainID.GNOSIS, ChainID.POLYGON_POS}
```

Comment:
you haven't added the decoder for polygon right? Please add it. I believe it is the same as gnosis

Reply:
As mentioned in discord polygon will go in another PR